_data/sites.json

```diff
@@ -5161,6 +5161,16 @@
             "danwin1210.me"
         ]
     },
+    {
```

Comment:
Argh, I meant to add a line break but without extra spaces, not remove the line break

Reply:
Can you please add the line break back, but without adding any space?